adapters/play/src/main/scala/caliban/PlayAdapter.scala

```diff
@@ -1,32 +1,35 @@
 package caliban
 
-import akka.stream.{ Materializer, OverflowStrategy, QueueOfferResult }
-import akka.stream.scaladsl.{ Flow, Sink, Source, SourceQueueWithComplete }
+import akka.stream.scaladsl.{Flow, Sink, Source, SourceQueueWithComplete}
+import akka.stream.{Materializer, OverflowStrategy, QueueOfferResult}
 import caliban.PlayAdapter.RequestWrapper
-import caliban.ResponseValue.{ ObjectValue, StreamValue }
+import caliban.ResponseValue.{ObjectValue, StreamValue}
 import caliban.Value.NullValue
 import caliban.interop.play.json.parsingException
 import play.api.http.Writeable
-import play.api.libs.json.{ JsValue, Json, Writes }
-import play.api.mvc.{ Action, ActionBuilder, AnyContent, PlayBodyParsers, Request, RequestHeader, Result, WebSocket }
+import play.api.libs.json.{JsValue, Json, Writes}
 import play.api.mvc.Results.Ok
+import play.api.mvc._
 import zio.Exit.Failure
-import zio.{ CancelableFuture, Fiber, IO, RIO, Ref, Runtime, Schedule, Task, URIO, ZIO }
 import zio.clock.Clock
 import zio.duration.Duration
+import zio.{CancelableFuture, Fiber, IO, RIO, Ref, Runtime, Schedule, Task, URIO, ZIO}
 
-import scala.concurrent.{ ExecutionContext, Future }
+import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Try
 
 trait PlayAdapter[R] {
 
+  val `application/graphql` = "application/graphql"
   def actionBuilder: ActionBuilder[Request, AnyContent]
   def parse: PlayBodyParsers
   def requestWrapper: RequestWrapper[R]
 
   implicit def writableGraphQLResponse[E](implicit wr: Writes[GraphQLResponse[E]]): Writeable[GraphQLResponse[E]] =
     Writeable.writeableOf_JsValue.map(wr.writes)
 
+  implicit val graphQLReqReader = Json.reads[GraphQLRequest]
```

Comment:
I don't think this is needed, there should already be a `Reads` available for `GraphQLRequest`

Reply:
You're right, is a leftover for the previous changes